problem_builder/public/js/mentoring.js

```diff
@@ -69,7 +67,14 @@ function MentoringBlock(runtime, element) {
 
     function renderAttempts() {
         var data = $('.attempts', element).data();
-        $('.attempts', element).html(attemptsTemplate(data));
+        if (data != undefined && data.max_attempts > 0) {
+           var message = _.template(
```

Comment:
Please note that template string is now pulled out of `mentoring_attempts.underscore` because it was using global `ngettext` but we require local problem builder `ngettext` for translation.